docs/developer/extending/apps/subscription-webhook-payloads.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Subscription Webhook Payloads
```

Comment:
We should include info about the version when we added it and also that this is a previe feature